pypiper/manager.py

```diff
@@ -771,7 +774,7 @@ def _set_status_flag(self, status):
         prev_status = self.status
         self.status = status
         self.pipestat.set_status(
-            record_identifier=self._pipestat_manager.sample_name,
+            record_identifier=self._pipestat_manager.cfg["record_identifier"],
```

Comment:
see above, can use `self._pipestat_manager.record_identifier`

Reply:
we should discourage using private variables outside of the package though -- is there another way to do this?

if this is really necessary, it indicates pipestat needs some other way to do this.

Reply:
It looks like this is simply that pypiper sets its pipestatmanager attribute as private:
https://github.com/databio/pypiper/blob/1c2c844dacf29c1d693af20e57f20e9cc1b60d4d/pypiper/manager.py#L345-L358

While the `record_identifier` attribute on the pipestatmanager is _not_ private.

Should we just change this to be `self.pipestat_manager = PipestatManger()` ?

Reply:
Oh I think I must have just misinterpreted things. I thought it was accessing a private variable from pipestat. If it's just accessing its own private variables, that's fine !